data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1520,56 +1578,97 @@ descriptions:new_tribe {
-            purpose = pgettext("building", "Smelts iron ore into iron and gold ore into gold.")
+            purpose = pgettext("building", "Smelts iron ore into iron and gold ore into gold."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Smelting Works
+            lore = pgettext("atlanteans_building", "We learned the smelting from our forefathers, a basic craftsmanship taming the fire and the ore into pure metals"),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Smelting Works
+            lore_author = pgettext("atlanteans_building", "A member of the craftsmans guild")
          }
       },
       {
          name = "atlanteans_toolsmithy",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Toolsmithy
-            purpose = pgettext("building", "Forges all the tools that your workers need.")
+            purpose = pgettext("building", "Forges all the tools that your workers need."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Toolsmithy
+            lore = pgettext("atlanteans_building", "People's power working for the benefit of all"),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Toolsmithy
+            lore_author = pgettext("atlanteans_building", "Faded sign found on top of this building")
          }
       },
       {
          name = "atlanteans_weaponsmithy",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Weapon Smithy
-            purpose = pgettext("atlanteans_building", "Forges tridents to equip the soldiers and to train their attack in the dungeon.")
+            purpose = pgettext("atlanteans_building", "Forges tridents to equip the soldiers and to train their attack in the dungeon."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site:  Weapon Smithy
+            lore = pgettext("atlanteans_building", "You hear that sound of metal stridend? The weapon smith just forged a trident!"),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Weapon Smithy
+            lore_author = pgettext("atlanteans_building", "Atlantean nursery rhyme")
+
          }
       },
       {
          name = "atlanteans_armorsmithy",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Armor Smithy
-            purpose = pgettext("atlanteans_building", "Forges shields that are used for training soldiers’ defense in the labyrinth.")
+            purpose = pgettext("atlanteans_building", "Forges shields that are used for training soldiers’ defense in the labyrinth."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Armor Smithy
+            lore = pgettext("atlanteans_building", "She twists and tweaks the metal into form, a soldier will it use as arm"),
```

Comment:
Doesn't rhyme…?

Reply:
Sorry must do this in DE: "Reim Dich oder ich fress dich".  Any better suggestions?

Reply:
She twists and tweaks the steel with might
that soldiers may therewith once fight
most skillful foes, whose deadly lance
from this their armour off will glance!

Reply:
who is she. atlantean workers are male I believe. especially a smith

Reply:
I'm not entirely sure about some Atlantean workers, but this close-up of the atl armoursmith's blender model looks vaguely female to me:
![grafik](https://user-images.githubusercontent.com/48293446/155400281-b1ad8553-7679-44b2-8660-7fcd20efe269.png) ![grafik](https://user-images.githubusercontent.com/48293446/155400369-9e966793-ff11-4f20-9518-acea49c91a62.png)

Reply:
I actually try to be gender neutral, but I think we must play it that hard.

Reply:
The *lance* here belongs to the *most skillful foe*, so this may reference a weapon from any tribe